### ISSUE TITLE
calcite-input type=number and calcite-input-number missing up/down labels

### ISSUE THREAD

**Issue by AdelheidF** [Esri/calcite-components#5287]
### Summary

calcite-input type=number and calcite-input-number are missing up/down aria labels.
Or maybe they should be hidden via aria-hidden.

![image](https://user-images.githubusercontent.com/44481292/189220407-9d2021c9-fc05-4f4d-93aa-f5bbf8889df2.png)

### Actual Behavior

missing up/down aria labels, maybe via new props

### Expected Behavior

 up/down aria labels or hidden

### Reproduction Sample

x

### Reproduction Steps

x

### Reproduction Version

beta.93

**Comment by AdelheidF** [comment]
looks good, tested with beta.93
